src/__tests__/Box.js

```diff
@@ -37,4 +37,9 @@ describe('Box', () => {
     expect(json).toMatchSnapshot()
     expect(json).toHaveStyleRule('background-color', theme.colors.green)
   })
+
+  test('align prop triggers warning', () => {
+    const json = renderer.create(<Box align="center" />).toJSON()
+    expect(json).toThrowErrorMatchingSnapshot()
```

Comment:
A prop type warning shouldn't throw an error, right?

Reply:
https://github.com/pricelinelabs/design-system/blob/master/src/Box.js#L46 it's returning an Error instance. i needed to replace missing coverage. i'm open to suggestions

Reply:
Right, the prop type definition function will return an Error, but React will only `console.warn` in this case, not throw, and only in development

Here's an example of mocking console in the FormField tests https://github.com/pricelinelabs/design-system/blob/2eaa75ffb651dbcd8ee364f262cfd7328b1ce9e1/src/__tests__/FormField.js#L9